docs/docs/client-source-without-graphql.md

```diff
@@ -0,0 +1,184 @@
+
+const ClientYAML = () => (
+  <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+    <h1>{YAMLData.title}</h1>
+    <div>
+      {YAMLData.content.map(data => {
+        return <div key={`content_item_${uuid.v4()}`}>{data.item}</div>
+      })}
+    </div>
+  </div>
+)
+export default ClientYAML
+```
+
+As you can see, the code above is nothing more, nothing less than a functional stateless React component, that when rendered by Gatsby, and without any extra configuration will display a page sourced from a YAML file.
+
+## JSON example
+
+In this part you'll use JSON as a datasource.
+
+### Adding the JSON content
+
+Create a folder named `content` if it doesn't exist, and then add a new file inside called `client-data.json` with the following content inside:
+
+```json
+{
+  "title": "JSON used in the client with React and Gatsby",
+  "content": [
+    {
+      "item": "Cupcake ipsum dolor. Sit amet marshmallow topping cheesecake muffin. Halvah croissant candy canes bonbon candy. Apple pie jelly beans topping carrot cake danish tart cake cheesecake. Muffin danish chocolate soufflé pastry icing bonbon oat cake. Powder cake jujubes oat cake. Lemon drops tootsie roll marshmallow halvah carrot cake."
+    },
+    {
+      "item": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    },
+    {
+      "item": 192.33
+    },
+    {
+      "item": 111111
+    }
+  ]
+}
+```
+
+### Creating the page
+
+Now that you have something that needs to be shown, all that's missing is a page to show it.
+
+Add a new file called `jsononclient.js` inside the `pages` folder with the following code:
+
+```javascript
+import React from "react"
+import uuid from "uuid"
+import JSONData from "../../content/client-data.json"
+const ClientJSON = () => (
+  <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+    <h1>{JSONData.title}</h1>
+    <div>
+      {JSONData.content.map(data => {
+        return <div key={`content_item_${uuid.v4()}`}>{data.item}</div>
+      })}
+    </div>
+  </div>
+)
+export default ClientJSON
+```
+
+The only thing different in this case, is the file import. Instead of the YAML file, this time you're importing directly a JSON file into the page component.
+
+Once again out of the box and without any extra configuration the page will show the content.
+
+## Joining the pieces
+
+After all these steps are complete, you should have the following file and folder structure:
+
+```
+  |gatsby-client-sourcing-YAML-JSON
+    |content
+      - client-data.yaml / - client-data.json
+    |src
+      |pages
+        - index.js
+        - 404.js
```

Comment:
there is no `404.js` from starter

Reply:
@muescha the file in question is there, as you can see below:

![muesha_404](https://user-images.githubusercontent.com/22988955/54707221-8568cb00-4b38-11e9-94b6-55f01ba57749.png)